src/source/vector_tile_source.js

```diff
@@ -155,6 +155,18 @@ class VectorTileSource extends Evented implements Source {
      *
      * @param {string[]} tiles An array of one or more tile source URLs, as in the TileJSON spec.
      * @returns {VectorTileSource} this
+     * @example
+     * map.addSource('vector_source_id', {
+     *     type: 'vector',
+     *     tiles: ['https://d25uarhxywzl1j.cloudfront.net/v0.1/{z}/{x}/{y}.mvt'],
```

Comment:
Where did this URL come from? Is this something we control? Maybe it'd be better to use a fake URL.